src/helpers/BapiClient.ts

```diff
@@ -211,6 +211,8 @@ type AddWishlistItemResponse =
 export enum AddToWhistlistFailureKind {
   OnlyOneParameterMustBeSet = 'OnlyOneParameterMustBeSet',
   ItemUnvailable = 'ItemUnvailable',
+  MaximumItemCountReached = 'MaximumItemCountReached',
+  VariantAlreadyPresent = 'VariantAlreadyPresent',
```

Comment:
As stated in our chat, let's consider renaming this to `ItemAlreadyPresent` since both variants and products can be added to a wishlist.

```suggestion
  ItemAlreadyPresent = 'ItemAlreadyPresent',
```